rust/osm2lanes/src/transform/tags_to_lanes/lane_count.rs

```diff
@@ -0,0 +1,375 @@
+use crate::locale::{DrivingSide, Locale};
+use crate::road::Designated;
+use crate::tag::{TagKey, Tags};
+use crate::transform::tags_to_lanes::{Infer, Oneway, RoadBuilder};
+use crate::transform::{RoadWarnings, TagsToLanesMsg};
+
+const LANES: TagKey = TagKey::from("lanes");
+const ONEWAY: TagKey = TagKey::from("oneway");
+
+#[derive(Debug)]
+pub(in crate::transform::tags_to_lanes) struct Counts {
```

Comment:
Ooh, never seen this fanciness before. Curious if `pub(super)` would be simpler? It could be a refactoring thing though; relative paths are more annoying when moving code around

Reply:
yeah, maintaining `pub(super)` becomes painful, especially when you need to refactor it to become `pub(super::super)`